File: client/src/app/_models/tweet.ts
export interface Tweet {
	id: string;
	text: string;
	attachments?: TweetAttachments;
	public_metrics?: TweetMetrics;
}

interface TweetAttachments {
	media_keys?: string[];
}

interface TweetMetrics {
	retweet_count: number,
	reply_count: number,
	like_count: number,
	quote_count: number
}

File: API/Entities/Tweet.cs
namespace API.Entities;

public class Tweet
{
    public string id { get; set; }
    public string author_id { get; set; }
    public string text { get; set; }
    public Attachments attachments { get; set; }
    public bool possibly_sensitive { get; set; }
    public PublicMetrics public_metrics { get; set; }
}